ui/components/src/ThresholdsEditor/ThresholdsEditor.tsx

```diff
@@ -85,6 +84,10 @@ export function ThresholdsEditor({ thresholds, onChange, hideDefault, disablePer
           draft.default_color = color;
         })
       );
+    } else {
+      onChange({
+        default_color: color,
+      });
```

Comment:
Would it help to add this to `handleThresholdColorChange` too?

Reply:
I don't think we need to because `handleThresholdColorChange` should only be called when `thresholds.steps` is already defined